modules/core/src/main/java/org/apache/ignite/internal/util/nio/GridNioServer.java

```diff
@@ -2510,20 +2511,10 @@ private void processSelectedKeysOptimized(SelectionKey[] keys) throws ClosedByIn
                     throw e;
                 }
                 catch (Exception | Error e) { // TODO IGNITE-2659.
```

Comment:
I think, if you would catch Throwable, it looks more pretty.

Reply:
I agree it's more correct to change to Throwable. However, catching Error explicitly shows that this code contains another error to be fixed. I would like to stay it as is. Also, IGNITE-2659 is added to our team's backlog as [IGN-4019](https://ggsystems.atlassian.net/browse/IGN-4019)